### ISSUE TITLE
err: exec: "docker-credential-osxkeychain"

### ISSUE THREAD

**Issue by clemenko** [acorn-io/acorn#479]
When testing Acron with `acorn run .` I am getting the following message. 

```
clempro:clemenko rke2 ( 147.182.179.229:6443 ) $ acorn run .
[+] Building 0.4s (3/3) FINISHED                                                                                                                                                                  
 => [internal] load build definition from acorn-dockerfile-3643591740                                                                                                                        0.2s
 => => transferring dockerfile: 65B                                                                                                                                                          0.2s
 => [internal] load .dockerignore                                                                                                                                                            0.2s
 => => transferring context: 2B                                                                                                                                                              0.2s
 => ERROR [internal] load metadata for docker.io/library/nginx:latest                                                                                                                        0.2s
------
 > [internal] load metadata for docker.io/library/nginx:latest:
------
Error: failed to solve: error getting credentials - err: exec: "docker-credential-osxkeychain": executable file not found in $PATH, out: 
```

Looks like I have an old `~/.docker/config.json` that might be causing the error. 

Acronfile:
```
containers: {
 web: {
  image: "nginx"
  ports: publish: "80/http"
 }
}
```

removing `~/.docker/config.json` fixed. 

cc: @ibuildthecloud

**Comment by cjellick** [comment]
from @ibuildthecloud in slack:

> I bet the buildkit client is looking for it. Can you put in an issue for this. I'll fix. If you do “acorn run” with a prebuilt image name this won't happen. But you obviously have a catch 22 in that you can’t build an image the first time.

**Comment by cjellick** [comment]
This should be addressed. moved all aspects of build to server side

**Comment by sangee2004** [comment]
@clemenko This issue should be addressed in the latest acorn version - `v0.6.0`.  Please feel free to reopen this issue , If you are still encountering this problem with the latest release.
